shared/js/shrinking_ui.js

```diff
@@ -493,6 +498,7 @@
    */
   ShrinkingUI.prototype._handleSendingStart =
     function su_handleSendingStart(evt) {
+      this.state.touch.initY = evt.touches[0].pageY;
```

Comment:
Comment to explain why we need this: for keeping the position the finger start to touch on the sliding cover, and we would use it to calculate the transition offset every frame.